### DISCUSSION TITLE
听说作者寄了

### DISCUSSION THREAD

**Discussion by ghost** [shuax/MouseInc.Settings#457] [General]
R.I.P，愿天堂没有社工库

**Comment by sumeco** [comment]
@sleepyW 你不会说话，就别评论。用着作者软件，跑来说风凉话，落井下石者，属实可恶，令人恶心作呕。

**Reply by ghost** [reply]
能恶心到你这种喜欢意淫的傻缺二极管真是太好了

**Reply by sumeco** [reply]
> 
言语间鄙夷无素养，想必家中长者已寄，无娘教，可怜。

**Reply by ghost** [reply]
生出来你这种智力缺陷二极管，你九泉之下的爹妈难辞其咎

**Comment by TensorABC** [comment]
作者怎么了？

**Reply by xmbai9507** [reply]
在QQ群热心给别人识别图中链接，结果链接里面有违法犯罪信息，所以被牵连判了三年然后缓刑，所以作者说是准备退网几年。真可惜了，希望作者日后能更好吧。

**Reply by rrtt2323** [reply]
> 在QQ群热心给别人识别图中链接，结果链接里面有违法犯罪信息，所以被牵连判了三年然后缓刑，所以作者说是准备退网几年。真可惜了，希望作者日后能更好吧。

woc，作者这是不是被人做局了。。。

**Reply by archonxp** [reply]
也不是热心给别人识别，是别人发了有违法犯罪信息的磁力链接（图片形式），作者用mouseINC的ocr识别了并且发出了文字形式的磁力链接问了一下内容，算转发也不算？总之是惹祸上身了

**Reply by WKGSSS** [reply]
按理说，作者并不知道链接内容啊，这么还算是呢，唉。听描述像是被套路了。

**Reply by Poqiu4396** [reply]
> 按理说，作者并不知道链接内容啊，这么还算是呢，唉。听描述像是被套路了。

太可惜了，作者自己也说了，不要滥用善心了，希望4年后耍哥归来吧

**Comment by Ctrl-Ling** [comment]
不会用RIP就别用

**Comment by leic4u** [comment]
这样说太不合时宜了，并不是所有的抖机灵都让人觉得你很幽默。愿你能做一个有情商的人。

**Comment by 578066229** [comment]
SB

**Comment by 567-su** [comment]
什么傻卵ylg，是你双亲趋势导致你这样的吗？

**Comment by 578066229** [comment]
什么，你双亲没了？难怪心里畸形哈哈，理解你了，不同情，活该




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "shuax/MouseInc.Settings"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年12月27日(星期二) 晚上11:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [shuax/MouseInc.Settings] 听说作者寄了 (Discussion #457)





 
什么傻卵ylg，是你双亲趋势导致你这样的吗？

**Comment by dmclj** [comment]
打从娘胎里出来就没人教你怎么好好说话吗？真可惜

**Comment by kyrie0817** [comment]
sb
